commercial/app/controllers/HostedContentController.scala

```diff
@@ -84,13 +84,13 @@ class HostedContentController(contentApiClient: ContentApiClient)
           val itemId = s"advertiser-content/$campaignName/$pageName"
           contentType match {
             case "video" =>
-              val trails = HostedTrails.fromContent(itemId, trailCount = 1, results)
+              val trails = HostedTrails.fromContent(itemId, trailCount = items.getOrElse(1), results)
               Cached(cacheDuration)(JsonComponent(hostedVideoOnward(trails.headOption)))
             case "article" =>
-              val trails = HostedTrails.fromContent(itemId, trailCount = 2, results)
+              val trails = HostedTrails.fromContent(itemId, trailCount = items.getOrElse(2), results)
```

Comment:
Could change this to `trailCount = <large number>` and then get rid of new `items` param.